src/main/java/cafeboard/board/BoardService.java

```diff
@@ -0,0 +1,45 @@
+package cafeboard.board;
+
+import jakarta.transaction.Transactional;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+@Service
+public class BoardService {
+
+    private final BoardRepository boardRepository;
+
+    public BoardService(BoardRepository boardRepository) {
+        this.boardRepository = boardRepository;
+    }
+
+    public BoardResponse save(BoardRequest boardRequest) {
+        Board board = boardRepository.save(new Board(boardRequest.boardName()));
+        return new BoardResponse(board.getBoardName(), board.getId());
+    }
+
+    public List<BoardResponse> findAll() {
+        return boardRepository.findAll()
+                .stream()
+                .map(board -> new BoardResponse(
+                        board.getBoardName(),
+                        board.getId()
+        )).toList();
+    }
+
+    @Transactional
+    public BoardResponse update(Long boardId, BoardRequest boardRequest) {
+        Board board = boardRepository.findById(boardId).orElseThrow(() ->
+                new NoSuchElementException("존재하지 않은 게시판입니다." + boardId));
+        board.setBoardName(boardRequest.boardName());
+        return new BoardResponse(board.getBoardName(), board.getId());
+    }
+
+    public void delete(Long boardId) {
+        Board board = boardRepository.findById(boardId).orElseThrow(() ->
+                new NoSuchElementException("존재하지 않은 게시판입니다." + boardId));
```

Comment:
요것도 사소한 거긴 한데, 한 줄에 점(`.`) 하나씩만 두시면 가독성 관점에서 더 좋다고 여겨지긴 해요
```suggestion
        Board board = boardRepository.findById(boardId)
                .orElseThrow(() -> new NoSuchElementException("존재하지 않은 게시판입니다." + boardId));
```